skribenten-backend/src/main/kotlin/no/nav/pensjon/brev/skribenten/model/Api.kt

```diff
@@ -155,6 +156,10 @@ object Api {
     data class SakContext(
         val sak: Pen.SakSelection,
         val brevmalKoder: List<String>,
+        val adressebeskyttelse: Boolean,
+        val vergemaalEllerFremtidsfullmakt: Boolean,
```

Comment:
Høyrast ut som ein enum for meg, men det er kanskje boolean vi får frå pdl?

Reply:
Jepp, men det er kanskje berre ekstra greier her når vi eigentleg ikkje treng det. Så viss av eller på er det vi treng å veta, så er det vel kurant nok å ha det her.